cmd/ocsp-updater/main_test.go

```diff
@@ -496,6 +496,10 @@ func TestMissingReceiptsTick(t *testing.T) {
 }
 
 func TestMissingOnlyReceiptsTick(t *testing.T) {
+	// Enable the ResubmitMissingSCTsOnly feature flag for this test run
```

Comment:
I had moved it while troubleshooting a unit test failure and forgot to move it back. Restored in https://github.com/letsencrypt/boulder/pull/2561/commits/67b325d685a3aaf087eb5b36f747af86ae1f1436